test/integration/raft_test.lua

```diff
@@ -0,0 +1,474 @@
+local fio = require('fio')
```

Comment:
This is a good test suite to start, however, these tests are not enough for testing failover. I would recommend adding  randomness to tests. For example, create a test where we write and read to cluster and periodically kill a leader. Data on cluster must be in a consistent state. Such test for Tarantool was very effective (see https://github.com/tarantool/tarantool/issues/5055#issuecomment-778166416).